API/hermes/hermes.h

```diff
@@ -188,12 +188,16 @@ class HERMES_EXPORT HermesRuntime : public jsi::Runtime {
   /// Unregister this runtime for sampling profiler.
   void unregisterForProfiling();
 
-  /// Register this runtime for execution time limit monitoring, with a time
-  /// limit of \p timeoutInMs milliseconds.
+  /// Asynchronously terminates the current execution.
```

Comment:
We should also mention that this may be invoked from any thread (most other API methods may not).